doc/API.md

```diff
@@ -131,12 +131,20 @@ As an alternative, it is possible to create a Libp2p instance with the construct
 
 ```js
 const Libp2p = require('libp2p')
+const PeerId = require('peer-id')
 
-// specify options
-const options = {}
+;(async () => {
```

Comment:
Perhaps we should do 

```js
async function main () {
  // ...
}

main()
```

This is a more generic approach than the async function being immediately called